src/main/java/teammates/common/datatransfer/questions/FeedbackRubricQuestionDetails.java

```diff
@@ -618,6 +624,7 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
 
         // Create table body
         StringBuilder tableBodyHtml = new StringBuilder();
+        StringBuilder tableBodyHtmlExcludingSelf = new StringBuilder();
```

Comment:
We have `tableBodyHtmlExcludingSelf` and `tableBodyExcludingSelfFragmentHtml`. I feel it would be good to standardise whether `ExcludingSelf` is before or after `Html`